apps/webClient/src/views/OurWork/index.tsx

```diff
@@ -103,20 +101,20 @@ const OurWork: React.FC<Props> = ({}) => {
           </Grid>
         </Grid>
         <Grid item>
-          <Box mt={2}>
+          <Box mt={2} mb={4}>
             <Grid container item justify="center">
-              <FlareIcon fontSize="large" />
+              <BulbIcon />
             </Grid>
           </Box>
         </Grid>
         <Grid item>
-          <Box mt={6}>
+          <Box mt={2}>
             <ServiceType />
           </Box>
-          <Box mt={6}>
+          <Box mt={2}>
             <ServiceType />
           </Box>
-          <Box mt={6} mb={6}>
+          <Box mt={2} mb={2}>
```

Comment:
I think this is already in master, please self review and rebase when you encounter this.

Reply:
No this is new. This was changed because the SVG's have some  white space so they're "bigger" than the MUI Icon placeholder that was there, so I adjusted this aswell. I should have pointed this out for this PR.